from_bad_to_good_function/from_bad_to_good_function.py

```diff
@@ -0,0 +1,20 @@
+import random
+
+
+def solution():
```

Comment:
It would be better to decompose the function into several smaller functions:
1. generate_pin()
2. replace_fives()
3. write_file()
Don't forget to add parameters to each function to provide more flexibility.
You can call all these methods in the __main__ section. 
Also, please add type hints to the functions